htlcswitch/circuit_map.go

```diff
@@ -61,6 +61,14 @@ type CircuitModifier interface {
 	// persistent references to a circuit. Returns a ErrUnknownCircuit if
 	// any of the incoming keys are not known.
 	DeleteCircuits(inKeys ...CircuitKey) error
+
+	// LookupCircuit queries the circuit map for the circuit identified by
+	// inKey.
+	LookupCircuit(inKey CircuitKey) *PaymentCircuit
+
+	// LookupOpenCircuit queries the circuit map for a circuit identified
+	// by its outgoing circuit key.
+	LookupOpenCircuit(outKey CircuitKey) *PaymentCircuit
```

Comment:
nit: Do these strictly belong in the circiut modifier if they are lookups? maybe just update the comment

Reply:
Created new interface